client/src/app/modules/components/sidebar/sidebar.component.html

```diff
@@ -23,6 +23,8 @@
            routerLink="/courses/{{ course.id }}/coverage-requests" [ngClass]="{'link-bold':activeLink == 4 && course.id==selectedCourse.id}">Coverage Requests</a>
         <a class="btn btn-light btn-sidebar input-block-level form-control link-style"
            routerLink="/courses/{{ course.id }}/settings" [ngClass]="{'link-bold':activeLink == 5 && course.id==selectedCourse.id}">Settings</a>
+        <a class="btn btn-light btn-sidebar input-block-level form-control link-style"
+           routerLink="/courses/{{ course.id }}/lessonplans" [ngClass]="{'link-bold':activeLink == 5 && course.id==selectedCourse.id}">Lesson Plans</a>
```

Comment:
should be `activeLink==6`

Reply:
is fixed in the code i have reserved, good catch. this is broken otherwise :cry: